.github/scripts/get_current_dev_tag.sh

```diff
@@ -2,4 +2,9 @@
 
 dev_tag=$(aws cloudformation describe-stacks --stack-name dev-ci --query "Stacks[0].Tags[?Key=='version'].Value" --output text)
 
+if [ -z "$current_deployed_tag" ]
+then
+     current_deployed_tag=v1.0.970-beta
```

Comment:
How is thi determined? Is it defined by the github runner environment? If it is, then I would prefer raising an error over using a currently deployed tag since we wouldn't want to push out something new if we're failing to check the old stuff.

Reply:
55c19723a834054b01457839aaf0ab13b0ca3fcd

It took me a minute to work out what was wrong with this. I'd been copying Ant's implementation from PSU, where he included this condition. It's effectively a backup value to ensure that confluence release notes are always updated correctly in the event that cloudformation fails to report the most recently deployed stack. I just used the version of the latest release.

Reply:
Aha, I see, that makes much more sense. However, are we sure we're happy with a fallback version number? In the (realistically, unlikely) event that we fail to update the version tag from cloudformation, I think it might be more sensible to have a reserved number, `v0.0.000-versionunknown` or something? So we don't tread on the toes of past releases.

Reply:
4028825c749a709b71b2efc6f1867071a0cadfa5